### DISCUSSION TITLE
3.6.1

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#54] [Announcements]
Blender 3.6.1, commit [c284cfcd0830](https://projects.blender.org/blender/blender/commit/c284cfcd0830).
Build environment:

    Visual Studio 2019, version 16.11.27
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/3.6.1'>3.6.1</a>.</em>

**Comment by dancingshoes** [comment]
Super BIG thankyou for doing this!!!!

**Comment by siamak89k** [comment]
hi dear.
first of all thank you.but I have got a serious problem with your some releases .
i opened  **3.6.1**and the elder  3.x.x your version you have released on win 7 ( **lenovo G510** : **CPU i7** -  **8GB RAM** -  **4GB AMD R7M256 Graphic that shared with Intel HD 4600** ) .but I can't see the Grid line . 

i am using blender.v2.9.3 and every thing is good and grid line is appeared.

what shall I do please help me bro .

**Reply by siamak89k** [reply]
watch this please video captured about my desktop 
 shall i install some app? like visual basic or something like that ?
https://github.com/nalexandru/BlenderCompat/assets/141816058/f5c4aa1d-9fa3-43c7-8f5f-163320784bb8

**Comment by Whallf** [comment]
Hello! I have a problem with launching release 3.6.1 and 3.6.0. When you start the application, a Windows window opens with the name Blender and a gray background. after a few seconds, it closes without a trace without any messages.
![2023-08-15_12-34-08](https://github.com/nalexandru/BlenderCompat/assets/137197762/b6225452-07fe-4617-9c49-1baa2d209a86)
